apps/rehabstod/src/pages/Patient/components/PatientHeader.tsx

```diff
@@ -1,41 +1,48 @@
-import { IDSIcon, IDSLink } from '@frontend/ids-react-ts'
-import { Link } from 'react-router-dom'
+import { IDSIcon, IDSButton } from '@frontend/ids-react-ts'
+import { useNavigate } from 'react-router-dom'
 import { Patient } from '../../../schemas/patientSchema'
 
-function countSickDays(startDate: Date, endDate: Date): number {
-  const today = new Date()
-  const daysSinceStart = Math.floor((today.getTime() - startDate.getTime()) / (1000 * 60 * 60 * 24))
-  const daysBeyondEnd = Math.max(0, Math.floor((today.getTime() - endDate.getTime()) / (1000 * 60 * 60 * 24)))
-  const daysSick = Math.max(0, daysSinceStart - daysBeyondEnd)
-  return daysSick
-}
-
 export function PatientHeader({ patient }: { patient: Patient }) {
+  const navigate = useNavigate()
+  const firstSickLeave = patient.sjukfallList[0]
+  const lastSickLeave = patient.sjukfallList[patient.sjukfallList.length - 1]
```

Comment:
I believe we only want an ongoing sickness. We can fetch that with the `apps/rehabstod/src/utils/isDateBeforeToday.ts` function from #823.

```typescript
const currentSickness = patient.sjukfallList.find(({ slut }) => !isDateBeforeToday(slut))
```